File: Linguagem C/Lista001_intro/Ex_003.c
//3) Fazer um algoritmo para converter uma temperatura de graus Celsius para graus Fahrenheit. A fórmula de
//conversão é: F = C * (9 / 5) + 32
#include <stdio.h>
int main ()
{   
    float Celsius, Faren;
    scanf("%f",&Celsius);

    Faren = 1.8*Celsius +32;
    printf("%0.2f", Faren);

return 0;

}

File: Python/001_Lista_intro/Ex_003.py
"""
3)  Fazer  um  algoritmo  para  converter  uma  temperatura  de  graus  Celsius  para  graus  Fahrenheit.  A  fórmula  de 
conversão é: F = C * (9 / 5) + 32
"""

print("Digite a temperatura em graus celsius\n")
temp = float(input())

fah= temp*(9/5) + 32
print("A conversao de graus celsius sera igual a: {:.2f} graus Fahrenheit".format(fah))